extension.yaml

```diff
@@ -192,32 +195,240 @@ params:
     default: subscribed
     required: true
 
-  - param: FIREBASE_MEMBER_TAGS_CONFIG
+  - param: MAILCHIMP_MEMBER_TAGS_WATCH_PATH
+    label: Firebase Member Tags Watch Path
+    description: >-
+      Specify the Firestore collection to listen for member tag changes
+
```

Comment:
I'm not sure all this formatting, or even the blank lines, will work here. Does anyone know? @jhuleatt @dackers86

Reply:
@dackers86 or @desoukya can you please provide a screenshot of what this looks like in the console and CLI install flows? We should double check that both can handle all the Markdown features being used in these descriptions

Reply:
If it helps, here's an example of how we phrase things in the `firestore-send-email` extension: https://github.com/firebase/extensions/blob/9f8d7fd6048bcaa7b5bc505cebe2e90494359e33/firestore-send-email/extension.yaml#L120-L128

Reply:
@jhuleatt Apologies for the late reply. Yes I tested this out before and the markdown rendered fine but noticed the JSON was not formatted. I even tried "```json ... ```" but that didn't work. I noticed however when installing locally using the terminal the JSON format is fine.